bob-apps/bin/bobp.rs

```diff
@@ -737,6 +772,12 @@ fn get_matches() -> ArgMatches<'static> {
                 .takes_value(false)
                 .long("verify"),
         )
+        .arg(
+            Arg::with_name("mode")
+                .help("keys in get operation are shuffled")
+                .takes_value(false)
```

Comment:
Now `mode` takes one of two values: `normal` or `random`, so the `true` should be passed. Additionally, default value should be specified